back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -45,6 +45,7 @@
       parameter :filter_can_moderate, 'Filter out the projects the current_user is not allowed to moderate. False by default', required: false
       parameter :filter_is_moderator_of, 'Filter out the publications the current_user is not moderator of. False by default', required: false
       parameter :filter_user_is_moderator_of, 'Filter out the publications the given user is moderator of (user id)', required: false
+      parameter :exclude_projects_in_included_folders, 'Exclude projects in included folders', required: false
```

Comment:
Might be worth saying that this is a boolean as it wasn't immediately obvious to me